scripts/process-tree.js

```
@@ -0,0 +1,570 @@
+}
+
+function defaultIsAlive(pid, processApi) {
+  try {
+    processApi.kill(pid, 0);
+    return true;
+  } catch (error) {
+    return error && error.code === 'EPERM';
+  }
+}
+
+function defaultGetProcessIdentity(pid, options = {}) {
```

ESLint

[error] Refactor this function to reduce its Cognitive Complexity from 22 to the 15 allowed.

(sonarjs/cognitive-complexity)